recipes/pysbol3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pysbol3" %}
+{% set version = "v1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SynBioDex/pySBOL3.git
+  git_rev: {{ version }}
```

Comment:
This project is nice enough to provide tarballs, like https://github.com/SynBioDex/pySBOL3/releases/download/v1.0.1/sbol3-1.0.1.tar.gz, do you mind change to it? Not only we save resources but we also make it more secure by checking the checksum of that tarball. Closing a tag/rev is not reproduceable b/c we can delete/modify them on GH.

Reply:
Hi @ocefpaf , the reason why I put git_url instead of a tarball is to be able to download the following **submodule** : https://github.com/SynBioDex/SBOLTestSuite/tree/ec49ef04da0252475c6e16dce17c85a0251cdd0a which is located in this **test directory**: https://github.com/SynBioDex/pySBOL3/tree/main/test . I've seen a same issue here: https://github.com/conda-forge/conda-forge.github.io/issues/619 . So, I followed this example: https://github.com/conda-forge/pyjion-feedstock/blob/main/recipe/meta.yaml . Then I used this command line in script session: `git submodule update --init --recursive` . Is there another way to use the tarball and to have access to this [SBOLTestSuite](https://github.com/SynBioDex/SBOLTestSuite) submodule ? Thank you!

Reply:
We can do exceptions in rare cases but ideally we should get the source from a stable tagged release. Testing stuff can be pulled/downloaded later at the testing phase if it is not in the source.

Reply:
Ok I downloaded the submodule from test section. Thank you :)